### ISSUE TITLE
SDK does not work with Looker-hosted instances on GCP

### ISSUE THREAD

**Issue by kgaspard** [looker-open-source/sdk-codegen#86]
Tested with AWS-hosted instance and it worked just fine.

looker.ini config:
```
[Looker]
# API version is required. 3.1 and 3.0 are currently supported. 3.1 is highly recommended. 
api_version=3.1
# Base URL for API. Do not include /api/* in the url 
base_url=https:/[clientName].gw1.cloud.looker.com
# API 3 client id 
client_id=xxxxxxxxxxxxxx
# API 3 client secret 
client_secret=xxxxxxxxxxxxxxxxxxxxxxxx
verify_ssl=false
```

**Comment by kgaspard** [comment]
To clarify the issue: I can't auth in. The following return no token:
`    const sdk = LookerNodeSDK.createClient()
    console.log('token: '+sdk.authSession._authToken.access_token)`

**Comment by jkaster** [comment]
@kgaspard If you open the GCP server in your browser, does the browser complain about the site not being secure?

**Comment by kgaspard** [comment]
It doesn't, the instance itself doesn't have an issue. Also, I believe it's not the fact that we host on GCP per se that's causing this, but rather that on our GCP instances the port to connect to the API is 443 (which may have different SSL requirements) and not 19999.

**Comment by jkaster** [comment]
Thanks for the answer. We'll continue looking into it.

**Comment by jkaster** [comment]
On the SDK side, this is closed via #91
